linters/taplo/plugin.yaml

```diff
@@ -97,7 +102,7 @@ lint:
       environment:
         - name: PATH
           list: ["${linter}"]
-      known_good_version: 0.8.0
+      known_good_version: release-cli-0.6.0
```

Comment:
I think this should be 0.8.0. Was this downgrade intentional?

Reply:
ah that was a mistake - merging and all that. fixed, thanks for the catch